gcloud-java-storage/src/main/java/com/google/gcloud/storage/BlobInfo.java

```diff
@@ -428,20 +420,20 @@ public ObjectAccessControl apply(Acl acl) {
     return storageObject;
   }
 
-  public static BlobInfo of(String bucket, String name) {
```

Comment:
This cause Travis to fail on some not up-to-date javadoc comments such as the one for signUrl